quickstarts/microsoft.mobilenetwork/mobilenetwork-update-packet-core-control-plane/azuredeploy.parameters.json

```diff
@@ -5,11 +5,23 @@
         "location": {
             "value": "eastus"
         },
-        "existingSiteName": {
-            "value": "GEN-PREREQ-existingSiteName"
+        "siteName": {
+            "value": "GET-PREREQ-siteName"
         },
-        "version": {
-            "value": ""
+        "existingMobileNetworkName": {
+            "value": "GET-PREREQ-existingMobileNetworkName"
+        },
+        "controlPlaneAccessInterfaceName": {
+            "value": "GEN-UNIQUE"
+        },
+        "controlPlaneAccessIpAddress": {
```

Comment:
seems like these last 3 param values should be output in the prereqs file - since that's how you're doing in the CUID?